spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreOperations.java

```diff
@@ -150,4 +152,13 @@
 	 * @return true if the given ID refers to an existing entity. False otherwise.
 	 */
 	<T> boolean existsById(Object id, Class<T> entityClass);
+
+	/**
+	 * Performs multiple read and write operations in a single transaction.
+	 * @param operations the function representing the operations to perform using a
```

Comment:
"a function that uses `DatastoreOpeations` to perform a series of operations in a transaction"

Reply:
@meltsufin changes made, PTAL.